chainercv/utils/testing/assertions/assert_is_bbox.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+
+def assert_is_bbox(bbox, size=None):
+    """Checks if bounding boxes satisfy bounding box format.
+
+    This function checks if given bounding boxes satisfy bounding boxe
+    format or not.
+    If the bounding boxes do not satifiy the format, this function raises an
+    :class:`AssertionError`.
+
+    Args:
+        bbox (~numpy.ndarray): Bounding boxes to be checked.
+        size (tuple of ints): The size of canvas.
+            If this argument is specified,
+            Each bounding box should be within the canvas.
+    """
+
+    assert isinstance(bbox, np.ndarray), \
+        'bbox must be a numpy.ndarray.'
```

Comment:
I always had this feeling that "should" is better than "must".
When there is "must" in an error message, I feel like I am scolded.